drivers/storage/portworx/component/pvccontroller.go

```diff
@@ -49,6 +51,12 @@ const (
 	AksPVCControllerInsecurePort = "10260"
 	// AksPVCControllerSecurePort is the PVC controller secure port.
 	AksPVCControllerSecurePort = "10261"
+
+	// K3sPVCControllerInsecurePort is the PVC controller port and health check port, due to the default port
+	// is already used on K3S we use different port for K3S.
+	K3sPVCControllerInsecurePort = "10260"
+	// K3SPVCControllerSecurePort is the PVC controller secure port.
+	K3sPVCControllerSecurePort = "10261"
```

Comment:
Instead of re-declaring the constants, let's just rename them and use one constant for the same ports.
May be `CustomPVCControllerSecurePort` and `CustomPVCControllerInsecurePort`.